### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
Reply-To email header configuration: https://github.com/mattermost/mattermost-webapp/pull/2327

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -1777,6 +1777,14 @@ So you don't miss messages, please make sure to change this value to an email yo
 | This feature's ``config.json`` setting is ``"FeedbackEmail": ""`` with string input.                                                                                 |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Notification Reply-To Address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Email address used in the Reply-To header when sending notification emails from Mattermost..
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: Wondering the use case for this setting? Is it primarily for Martin's email plugin?

Comment (wiersgallak) [reply source/administration/config-settings.rst]: Yes that is why it was added. (Fixed the dbl period issue too)

Comment (jasonblais) [reply source/administration/config-settings.rst]: Wondering if the plugin would be ready to mention here as a sample use case? https://github.com/DSchalla/mailermost-plugin

Comment (wiersgallak) [reply source/administration/config-settings.rst]: @mkraft - I'd defer that question to you :)

Comment (wiersgallak) [reply source/administration/config-settings.rst]: I'll add a note to follow up on this afterwards.

**Pull request merged**